src/mlpack/core/data/detect_file_type.cpp

```diff
@@ -47,7 +47,12 @@ std::string GetStringType(const arma::file_type& type)
  * from Armadillo's function guess_file_type_internal(), but we avoid using
  * internal Armadillo functionality.
  *
+ * If the file is detected as a CSV, and the CSV is detected to have a header
+ * row, the stream `f` will be fast-forwarded to point at the second line of the
+ * file.
+ *
  * @param f Opened istream to look into to guess the file type.
+ * @param filename Name of file, for output purposes.
```

Comment:
There is no filename paramter.

Reply:
Thanks!  Fixed in 487ab565.  (The comment was an artifact of some refactoring I did... but later undid.)